src/pages/carte.tsx

```diff
@@ -34,6 +34,7 @@ export const layerURLKeysToMapConfigPath = {
   besoinsEnChaleur: 'besoinsEnChaleur',
   besoinsEnFroid: 'besoinsEnFroid',
   besoinsEnChaleurIndustrieCommunes: 'besoinsEnChaleurIndustrieCommunes',
+  communesFortPotentielPourCreationReseauxChaleur: 'communesFortPotentielPourCreationReseauxChaleur.show',
 } as const satisfies { [key: string]: MapConfigurationProperty<boolean> };
```

Comment:
possible d'avoir key: keyof typeof tilesInfo?

Reply:
tilesInfo non car c'est des sources de données et ici on veut des layers (il y a parfois plusieurs layers pour une même source)

Par contre j'y ai pensé à utiliser par exemple le 1er niveau de MapConfiguration. Mais typescript m'emmerde avec `An index signature parameter type cannot be a literal type or generic type. Consider using a mapped object type instead.`...